resources/js/Pages/Adminland/Hardware/Create.vue

```diff
@@ -153,14 +151,10 @@ export default {
   },
 
   methods: {
-    updateStatus(payload) {
-      this.form.lend_hardware = payload;
-    },
-
     submit() {
       this.loadingState = 'loading';
-      if (this.form.employee_id) {
-        this.form.employee_id = this.form.employee_id.value;
+      if (!this.form.lend_hardware) {
+        this.form.employee_id = null;
```

Comment:
@djaiss  this is to be sure not to select an employee if you unselect the checkbox later